lib/widgets/app_bars/searchable_app_bar_style.dart

```diff
@@ -10,7 +10,7 @@ class SearchableAppBarStyle {
       );
 
   static double toolbarHeight(BuildContext context) =>
-      responsive.isMobile(context) ? 48 : 56;
+      responsive.isMobile(context) ? 48 : 112;
```

Comment:
IMO, it will be side effect for some other screen. WDYT?

Reply:
This style is only used in searchable_app_bar. It should always have the same widgets in the app bar IMO

Reply:
I saw `searchable_app_bar` is used in: `contacts_selection_view`, `new_private_chat_view`, `share_view`. Can you demo it too?

Reply:
Done, please check description and new commit

Reply:
but please click on the search to show the detail when search available. thank you